.github/workflows/ci.yaml

```diff
@@ -49,7 +49,9 @@ jobs:
         run: bazel --batch test $(bazel query "//src/test/..." | grep "Test$") --noshow_progress --test_output=errors
 
       - name: Integration Tests
-        run: bazel --batch test //test/integration:asset //test/integration:credentials //test/integration:iam //test/integration:kms //test/integration:logging //test/integration:pubsub //test/integration:redis //test/integration:library --noshow_progress
+        run: |
+          bazel clean --expunge --async
+          bazel --batch test //test/integration:asset //test/integration:credentials //test/integration:iam //test/integration:kms //test/integration:logging //test/integration:pubsub //test/integration:redis //test/integration:library --noshow_progress
```

Comment:
Why is each integration tests mentioned individually? Why not just `bazel test //...` for all tests? This would greatly simplify CI & workstation consistency between the tests and will not need any extra knowledge to maintain it (basically I have forgotten to add compute test in this list).

If there is a need to launch integration tests as a separate entity (to separate them from unit tests), please consider using [test_suite](https://docs.bazel.build/versions/main/be/general.html#test_suite) instead of mentioning each test individually.

We also need to add compute tests here, either as part of this PR or separately (I'll add it, if you prefer it to be done separately).

Reply:
Because `bazel test //...` runs all the update (i.e. non-test) targets as well. `test_suite` looks like a good option, will explore this separately.

Adding compute tests SGTM, done.

Reply:
bazel test //... will run only what is marked as `Test = true` in bazel [rule](https://docs.bazel.build/versions/4.1.0/skylark/lib/globals.html#parameters-30) definition. 
The integration tests are defined subject for `bazel test` https://github.com/googleapis/gapic-generator-java/blob/master/rules_bazel/java/integration_test.bzl#L87, `_update` targets are not https://github.com/googleapis/gapic-generator-java/blob/master/rules_bazel/java/integration_test.bzl#L164

Reply:
bazel test //... will run only what is marked as `Test = true` in bazel [rule](https://docs.bazel.build/versions/4.1.0/skylark/lib/globals.html#parameters-30) definition. 
The integration tests are defined subject for `bazel test` https://github.com/googleapis/gapic-generator-java/blob/master/rules_bazel/java/integration_test.bzl#L87, `_update` targets are not https://github.com/googleapis/gapic-generator-java/blob/master/rules_bazel/java/integration_test.bzl#L164

Reply:
Done.